cas-server-webapp/src/main/webapp/WEB-INF/view/jsp/protocol/3.0/casServiceValidationSuccess.jsp

```diff
@@ -49,9 +49,6 @@
                         <cas:${fn:escapeXml(attr.key)}>${fn:escapeXml(attrval)}</cas:${fn:escapeXml(attr.key)}>
                     </c:forEach>
                 </c:forEach>
-                <cas:isFromNewLogin>${fn:escapeXml(isFromNewLogin)}</cas:isFromNewLogin>
```

Comment:
Conceptually, I feel it's two different blocks (user attributes != authentication attributes). The easiest transition seems to use the existing user attributes to return more information, but in any case, the client must be aware of what is the attribute for so CAS clients should be upgraded to deal with these new attributes, whether they are user attributes or not.